src/Components/Nav/Nav.js

```diff
@@ -95,7 +93,11 @@ class Nav extends Component {
               <NavHeart />
               <NavCs />
               <button
-                className={this.state.navSearchBtnActive ? "navSearchBtnChanged" : "navSearchBtn"}
+                className={
+                  this.state.navSearchBtnActive
+                    ? "navSearchBtnChanged"
+                    : "navSearchBtn"
```

Comment:
```suggestion
                    ? "navSearchBtn changed"
                    : "navSearchBtn"
```
위와 같이 클래스명을 분리해서 적어주면 공통 요소는 `navSearchBtn`를 통해서 적용해주고 변화하는 부분만 changed를 통해 스타일링 할 수 있습니다.